src/dagmc.cpp

```diff
@@ -211,15 +230,44 @@ void DAGUniverse::init_geometry()
     if (mat_str == "graveyard") {
       graveyard = vol_handle;
     }
-
     // material void checks
     if (mat_str == "void" || mat_str == "vacuum" || mat_str == "graveyard") {
       c->material_.push_back(MATERIAL_VOID);
     } else {
       if (uses_uwuw()) {
         uwuw_assign_material(vol_handle, c);
```

Comment:
How do we want the overrides to work when UWUW materials are present? I think that we'd want them to override UWUW material assignments as well.

Reply:
I did something about this:

first check if override is path by cell ID, 
then check if uwuw is used to recover the material name (as override key), 
finally use the mat_str as override key


in all 3 cases, use to legacy assignment to override mat assignment.

Reply:
@pshriwise I left this one open so you can make sure this fit what you had in mind.